src/test/java/testsuite/integration/container/AuroraMysqlIntegrationBaseTest.java

```diff
@@ -92,7 +129,24 @@
   protected static Proxy proxyReadOnlyCluster;
   protected static final Map<String, Proxy> proxyMap = new HashMap<>();
 
+  protected String[] instanceIDs; // index 0 is always writer!
+  protected int clusterSize = 0;
+
   protected final ContainerHelper containerHelper = new ContainerHelper();
+  protected static final AmazonRDS rdsClient = AmazonRDSClientBuilder
+      .standard()
+      .withRegion(DB_REGION)
+      .withCredentials(new DefaultAWSCredentialsProviderChain())
```

Comment:
Need to check what are the other options to init aws client. Ideally we'd like to use an auth token, not user id and not secret key